web/views/member_view.ex

```diff
@@ -1,3 +1,18 @@
 defmodule Community.MemberView do
   use Community.Web, :view
+  @gravatar_host "https://secure.gravatar.com/avatar/"
+
+  def gravatar(email) do
+    hashed_email = email
+    |> String.downcase
+    |> String.strip
+    |> md5_hash
+
+    @gravatar_host <> hashed_email
+  end
+
+  defp md5_hash(email) do
+    :crypto.hash(:md5, email)
+    |> Base.encode16(case: :lower)
```

Comment:
It could but I in general avoid the one liner. I don't really see a benefit and if I decide it needs another pipe between or after, it's easier to add.